app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/teidata/TEIDataModule.kt

```diff
@@ -184,4 +189,12 @@ class TEIDataModule(
 
     @Provides
     fun provideDateUtils() = DateUtils.getInstance()
+
+    @Provides
+    @PerFragment
+    fun providesOURepository(
+        d2: D2,
+    ): OUTreeRepository {
+        return OUTreeRepository(OURepositoryConfiguration(d2, OrgUnitSelectorScope.ProgramCaptureScope(programUid!!)))
```

Comment:
This will cause a crash when the programUid is null. That happends in the all enrollments screen